python/doc/theory/meta_modeling/polynomial_least_squares.rst

```diff
@@ -3,95 +3,102 @@
+where :math:`P` denotes the number of terms, which is equal to
+:math:`(n_X + 1)` (resp. to :math:`(1 + 2n_X + n_X (n_X - 1)/2)`) when
+using a linear (resp. a quadratic) approximation, and the family
+:math:`(\psi_j,j=0,\dots,P-1)` gathers the constant monomial
+:math:`1`, the monomials of degree one :math:`x_i` and possibly the
+cross-terms :math:`x_i x_j` as well as the monomials of degree two
+:math:`x_i^2`. Using the vector notation
+:math:`\vect{a}  =  (a_{0} , \dots , a_{P-1} )^{\textsf{T}}`
+and
+:math:`\vect{\psi}(\vect{x})  =  (\psi_{0}(\vect{x}) , \dots , \psi_{P-1}(\vect{x}) )^{\textsf{T}}`,
+this rewrites:
 
   .. math::
 
-      \underline{y} \, \, \approx \, \, \widehat{h}(\underline{x}) \, \, = \, \, \underline{a}^{\textsf{T}} \; \underline{\psi}(\underline{x})
+      y  \approx  \widehat{\model}(\vect{x})  =  \vect{a}^{\textsf{T}} \vect{\psi}(\vect{x})
 
-| A *global* approximation of the model response over its whole
-  definition domain is sought. To this end, the coefficients :math:`a_j`
-  may be computed using a least squares regression approach. In this
-  context, an experimental design
-  :math:`\underline{\cX} =(x^{(1)},\dots,x^{(N)})`, i.e. a set of
-  realizations of input parameters is required, as well as the
-  corresponding model evaluations
-  :math:`\underline{\cY} =(y^{(1)},\dots,y^{(N)})`.
+A *global* approximation of the model response over its whole
+definition domain is sought. To this end, the coefficients :math:`a_j`
+may be computed using a least squares regression approach. In this
+context, an experimental design that is, a set of observations of
+input parameters, is required:
 
-| The following minimization problem has to be solved:
+.. math::
 
-  .. math::
+    \cX = \left\{ \vect{x}^{(1)}, \dots, \vect{x}^{(N)} \right\},
 
-     \mbox{Find} \quad \widehat{\underline{a}} \quad \mbox{that minimizes} \quad \cJ(\underline{a}) \, \, = \, \, \sum_{i=1}^N \; \left( y^{(i)} \; - \; \underline{a}^{\textsf{T}}  \underline{\psi}(\underline{x}^{(i)}) \right)^2
+as well as the corresponding model evaluations:
 
-  The solution is given by:
+.. math::
 
-  .. math::
+    \cY = \left\{ y^{(1)},\dots,y^{(N)} \right\}.
 
-      \widehat{\underline{a}} \, \, = \, \, \left( \underline{\underline{\Psi}}^{\textsf{T}} \underline{\underline{\Psi}}  \right)^{-1} \; \underline{\underline{\Psi}}^{\textsf{T}}  \; \underline{\cY}
+The following minimization problem has to be solved:
 
-  where:
+.. math::
 
-  .. math::
+    \mbox{Find} \quad \widehat{\vect{a}} \quad \mbox{that minimizes} \quad \cJ(\vect{a})  =  \sum_{i=1}^N \left( y^{(i)} - \vect{a}^{\textsf{T}}  \vect{\psi}\left(\vect{x}^{(i)}\right) \right)^2
+
+Let :math:`\vect{y} = (y^{(1)},\dots,y^{(N)})^T \in \Rset^{N}` be the
+vector of output observations.
+The solution is given by the normal equations:
+
+.. math::
+
+    \widehat{\vect{a}}  =  \left( \boldsymbol{\Psi}^{\textsf{T}} \boldsymbol{\Psi}  \right)^{-1} \boldsymbol{\Psi}^{\textsf{T}}  \vect{y}
```

Comment:
Here and elsewhere `boldsymbol` should be replaced with `mat`

Reply:
You are right. Even more: this is a matrix and the `\mat` macro from maths_notations.sty is for that specific purpose. By the way, the `{\textsf{T}` can be replaced with the `\Tr` macro. 

Fixed at 2d79cecad.